packages/eslint-plugin-nimble/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ni-private/eslint-plugin-nimble",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Common ESLint configuration for the Nimble repository",
+  "repository": {
```

Comment:
`package.json` is always included by default, and none of our other `package.json` include this self reference. `index.js` is also auto-included because of it being the default for the `main` module name. Why add these explicitly?

> Edit: in the other comment I suggest different files being exported so this should reflect that

I don't see another comment mentioning exported files.

Reply:
fair, double-checked and I was mixing up the `files` prop and the [`exports`](https://nodejs.org/api/packages.html#package-entry-points) prop where `package.json` has to be mentioned manually.

It's probably a good idea to add an exports as well (`files` is what is included in the tgz and `exports` is what is allowed to be resolved from the package when imported).